src/main.c

```diff
@@ -252,8 +252,8 @@ gboolean vb_load_uri(Client *c, const Arg *arg)
         rp  = realpath(path, NULL);
         uri = g_strconcat("file://", rp, NULL);
         free(rp);
-    } else if (strchr(path, ' ') || !strchr(path, '.')) {
-        /* use a shortcut if path contains spaces or no dot */
+    } else if (!strchr(path, '.')) {
```

Comment:
Is this really needed for the parameter free shortcuts? I've added the `strchr(path, ' ')` to allow to search for something containing dots like "document.activeElement". This would not work without the check for space.